README.md

```diff
@@ -1,24 +1,24 @@
-AChecker
-========
+# AChecker
 
 AChecker is an automated accessibility checker used to evaluate the accessibility of HTML pages, and help ensure they can be accessed by all individuals, including those with disabilities, using assistive technologies to navigate the Internet.
 
 AChecker live site: http://achecker.ca
 
 What sets AChecker apart from other automated accessibility checkers?
 
-1. Reviewers can interact with the system to make decisions on potential barriers that automated checkers can not determine with certainty.
-2. Choose from a range of accessibility standards to review conformance with various international accessibility requirements.
-3. Design custom accessibility guidelines tailored specifically to your organization
-4. View existing guidelines in AChecker to see exactly what it is reviewing.
-5. Design new accessibility checks and have them added to AChecker.
+- Reviewers can interact with the system to make decisions on potential barriers that automated checkers can not determine with certainty.
+- Choose from a range of accessibility standards to review conformance with various international accessibility requirements.
+- Design custom accessibility guidelines tailored specifically to your organization
+- View existing guidelines in AChecker to see exactly what it is reviewing.
+- Design new accessibility checks and have them added to AChecker.
 
-Installation
-------------
+## Installation
 
-1. Unzip the achecker.tar.gz file into a php enabled, web accessible directory.
-2. Open the installer at AChecker/install
-3. Follow the instruction provided by the installer.
+- Clone this repository
+- Be sure to have Composer [setup on your system/server](https://getcomposer.org/download/)
+- Run `composer update` to install dependencies
```

Comment:
Running `composer update` returns this error: `-bash: composer: command not found`

Reply:
Install the composer first [b]Be sure to have Composer[/b].
https://getcomposer.org/download/

Reply:
I did install the composer first, then I follow this instruction: https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies to install project dependencies.

Reply:
Great. It reinforces that we do should have a system requirement section.